src/add/observable/combineLatest.ts

```diff
@@ -1,6 +1,4 @@
 import {Observable} from '../../Observable';
 import {combineLatestStatic} from '../../operator/combineLatest';
 
-Observable.combineLatest = combineLatestStatic;
-
-export var _void: void;
```

Comment:
Let's go that way. I don't think this need to block this whole PR. (export void if empty, remove if it's unnecessary)